pkg/upgrade/control_plane.go

```diff
@@ -261,6 +261,12 @@ func (u *ControlPlaneUpgrader) updateMachines(machines *clusterapiv1alpha1.Machi
 
 	// TODO add more error logs on failure conditions
 	for _, machine := range machines.Items {
+		annotations := machine.GetAnnotations()
+		// Skip any machine that already has the annotation we're looking for
+		if val, ok := annotations["upgrade-id"]; ok && val == u.upgradeID {
```

Comment:
Let's move the name of the annotation to a constant. We should also namespace it in some way.

Reply:
`com.vmware.cluster-api-upgrade-tool.upgrade-id` ? i have always been bad at these java style name spacing 😆

Reply:
Eh, let's just do a constant for now - ok to keep it as `upgrade-id`. If we feel like it, we can file an issue after this is merged to adjust the label key later.